### PR TITLE
Add doc build steps

### PR DESCRIPTION
**Pull request by dedemorton**
Not sure where to put these steps in the readme.

### CODE REVIEW

**Comment by jakelandis** [comment]
This looks good... a couple questions. 

Should we also include instructions to build the plugin documentation ?

Should we call the need for asciidoctor to be installed ?

**Comment by dedemorton** [comment]
Do you mean instructions for _generating_ the plugin documentation? The plugins get built when you run the local doc build. IMO, the directions for generating the plugin docs should be covered in the directory where we keep tool rather than here (maybe we can point to them from here). That's true of the versioned plugin reference, too. No one should update the versioned plugin reference *except* the person who is generating the files for it (eventually that will be automated). The main reason for adding the instructions here is so that core contributors can build the open source docs locally.

**Comment by dedemorton** [comment]
@jakelandis We don't use asciidoctor. We might revisit that decision later. Bear in mind that asciidoctor supports some things that asciidoc does not.

File: README.md [hunk 1]
```diff
@@ -120,6 +120,26 @@ Example:
...
+Check out `master` in the `docs` repo.
+
+Run the doc build script from within the `docs` repo. For example:
+
+```
+./build_docs.pl --doc ../logstash/docs/index.asciidoc --chunk=1 -open
+```
+
+To build the Logstash Reference with X-Pack content, see
+the [x-pack-logstash readme]https://github.com/elastic/x-pack-logstash#building-documentation.
```
Comment (jakelandis) [inline README.md]: This is a private repo, so we shouldn't link to it here.

File: README.md [hunk 2]
```diff
@@ -120,6 +120,26 @@ Example:
 
 Drip does not work with STDIN. You cannot use drip for running configs which use the stdin plugin.
 
+## Building Logstash documentation
+
+To build the Logstash Reference on your local machine, clone the following repos:
+
+[logstash]https://github.com/elastic/logstash - contains main docs about core features
```
Comment (jakelandis) [inline README.md]: this links need parenthesis around them [name]`(`https://...`)`

File: README.md (hunk 1 above)
Comment (dedemorton) [reply README.md]: good point

File: README.md (hunk 2 above)
Comment (dedemorton) [reply README.md]: good catch! meant to change them before pushing. :-/

**Comment by dedemorton** [comment]
@jakelandis I've pushed fixes. Thanks!

**Comment by jakelandis** [comment]
one more minor change ... markdown requires a blank line for the new line to be honored.

(specifically between the 3 repo's):
image

**Comment by dedemorton** [comment]
@jakelandis Thanks, added the blank lines. Ready to merge?

**Comment by jakelandis** [comment]
LGTM

**Pull request closed without merging**